### ISSUE TITLE
[ToolBar] Error is thrown when the ToolBar is focused if the first tool is a dropdown button or a split button

### ISSUE THREAD

**Issue by siyka-marinova** [telerik/kendo-angular#2818]
**Describe the bug**
Error is thrown when the ToolBar is clicked or focused with the keyboard if the first tool is a dropdown button or a split button.

**To Reproduce**
1. Open https://stackblitz.com/edit/angular-p9zzyq
2. Click somewhere on the toolbar, but not on a tool 
OR  click inside the input and press Tab two times to focus the toolbar

Result: the following error is thrown:
```
ERROR TypeError: Cannot read property 'type' of undefined
    at ToolBarSplitButtonComponent.focus (toolbar-splitbutton.component.ts?301d:236)
    at NavigationService.focusFirst (navigation.service.ts?3f22:96)
    at ToolBarComponent.onFocus (toolbar.component.ts?e1ab:224)
    at Object.eval [as handleEvent] (AppComponent.html:16)
    at handleEvent (core.umd.js?e9eb:10282)
    at callWithDebugContext (core.umd.js?e9eb:11375)
    at Object.debugHandleEvent [as handleEvent] (core.umd.js?e9eb:11078)
    at dispatchEvent (core.umd.js?e9eb:7741)
    at eval (core.umd.js?e9eb:8185)
    at HTMLElement.eval (platform-browser.umd.js?8439:1068)
    at ZoneDelegate.invokeTask (zone.js?d135:423)
    at Object.onInvokeTask (core.umd.js?e9eb:3881)
    at ZoneDelegate.invokeTask (zone.js?d135:422)
    at Zone.runTask (zone.js?d135:195)
    at ZoneTask.invokeTask [as invoke] (zone.js?d135:498)
    at invokeTask (zone.js?d135:1744)
    at HTMLElement.globalZoneAwareCallback (zone.js?d135:1770)
```

**Browser**
 - OS: Windows, Mac
 - Browser: all

**Comment by siyka-marinova** [comment]
Fix for this issue is available in the [latest dev. version](https://www.telerik.com/kendo-angular-ui/components/installation/development-builds/#toc-obtaining-development-builds) of `kendo-angular-toolbar`

**Comment by Raisolution** [comment]
Available in `v3.1.2`
